cypress/integration/login.js

```diff
@@ -20,5 +20,8 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 1)
+        var d = new Date();
+        var currentday = d.getDate();
+        cy.get('[data-test="timesheets-current-day"]').contains(currentday)
```

Comment:
Is this locator [data-test="timesheets-current-day"] unique?

Reply:
changed to `cy.get('.calendar__day.calendar--today')`